docs/integrations/sources/hubspot-migrations.md

```diff
@@ -1,5 +1,13 @@
 # HubSpot Migration Guide
 
+## Upgrading to 3.0.0
+
+:::note
+This change is only breaking if you are syncing the Marketing Emails stream.
+:::
+
+This update brings extended schema with data types changes for the Marketing Emails stream.
```

Comment:
This update brings extended schema with data type changes for the Marketing Emails stream.

Reply:
please give them the appropriate migration guide steps: https://docs.google.com/document/d/1V3ybA24Lj_fy7IQwsVVzSxs46qGOsUydwlAQEYSQfcE/edit?pli=1

Reply:
Updated